### PR TITLE
ovito: 3.3.1 -> 3.3.5

### PR DESCRIPTION
**Pull request by siraben**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/ovito/default.nix [hunk 1]
```diff
@@ -15,26 +15,25 @@
...
 
   propagatedBuildInputs = with python.pkgs; [ sphinx numpy sip pyqt5 matplotlib ase ];
 
-  enableParallelBuilding = true;
-
   meta = with stdenv.lib; {
     description = "Scientific visualization and analysis software for atomistic simulation data";
     homepage = "https://www.ovito.org";
-    license = licenses.gpl3;
+    license = licenses.gpl3Only; # dual-licensed under GPL3Only and MIT
```
Comment (prusnak) [inline pkgs/development/python-modules/ovito/default.nix:36]:
```suggestion
    license = with licenses; [ gpl3Only mit ];
```

**Review by prusnak — CHANGES_REQUESTED** [review]

**Review by prusnak — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Comment by SuperSandro2000** [comment]
Running nixpkgs-review on this froze my machine almost three times cause it used more than 12GB of RAM...

**Comment by siraben** [comment]
So sorry! I think it would be better off if someone else tried upgrading it, as I'm not familiar with this package.

**Comment by SuperSandro2000** [comment]
I build python38Packages.ovito successfully which needs to be enough for now.

**Pull request closed without merging**